great_expectations/render/__init__.py

```diff
@@ -225,9 +225,9 @@ def clean_null_attrs(self, data: dict, **kwargs: dict) -> dict:
         their values are None."""
         data = deepcopy(data)
         for key in RenderedAtomicValueSchema.REMOVE_KEYS_IF_NONE:
-            if key == "graph" and key in data and data[key]["graph"] is None:
+            if key in data and data[key] is None:
```

Comment:
```suggestion
          if key == "graph" and key in data and data.get(key, {}).get("graph") is None:
```


Does this work (assuming the other change was reverted)? It would make for an easier-to-follow blame if it does.